cmd/tempo-serverless/cloud-run/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# 
+#  docker run -p 8080:8080 tempo-serverless
+#
+#  to exercise the function
+#    curl http://localhost:8080/?start=1000&end=1001&...
+#
+
+#
+# build the serverless container image
+#
+#  todo: FROM scratch saves ~5MB which could be meaningful in a serverless setting, but using scratch gave strange errors on query.
+FROM alpine:3.15 as certs 
```

Comment:
I think 3.16 is out if you want to upgrade.

Reply:
Not a bad idea, but all other images are on 3.15. I'm going to leave this on 3.15 as well. We should open a separate PR that moves all of them forward as one.